report.py

```diff
@@ -0,0 +1,57 @@
+from pathlib import Path
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sbn
+import json
+
+
+class ReportAssistant():
+    def __init__(self):
+        self.usage = pd.read_csv("./state/metrics/usage.csv")
+        # self.messages = pd.read_csv("./state/metrics/messages.csv")
+        self.feedback = pd.read_csv("./state/metrics/feedback.csv")
```

Comment:
These paths should be injected. Better yet, perhaps we modify the MetricsHandler class to have getter methods for the information. Then ReportAssistant gets a MetricsHandler and asks it for the data.

If we move to a SQL backend (or any other backend) for the metrics, we don't want ReportAssistant to need to change.